test/index.js

```diff
@@ -1,5 +1,62 @@
 const tap = require('tap');
 
-tap.test('Error case', test => {
-    test.done();
-});
+const headLinesJson = require('./fixtures/getHeadlines.json');
+const opinionJson = require('./fixtures/getOpinionOn.json');
+const reviewJson = require('./fixtures/getReview.json');
+
+var lambda = require('../src/index').handler;
+
+tap.test('Test get headlines intent', test => {
+	test.plan(2);
+
+	lambda(
+		headLinesJson, {
+			succeed: function (response) {
+				test.equal(response.sessionAttributes.lastIntent, "GetHeadlines");
```

Comment:
I've just realized that `test.match` works with objects too, so you can write

```js
test.match(response, {
   sessionAttributes: {
      lastIntent: 'GetHeadlines'
   }
})
```

this doesn't blow up when the response is not what you expect and gives you a better output in the test result